sdk/monitor/azure-monitor-query/azure/monitor/query/_exceptions.py

```diff
@@ -27,15 +27,14 @@ class LogsQueryError:
     """A human readable error message."""
     details: Optional[List[JSON]] = None
     """A list of additional details about the error."""
-    status: LogsQueryStatus
+    status: Literal[LogsQueryStatus.FAILURE]
     """Status for error item when iterating over list of results. Always "Failure" for an instance of a
     LogsQueryError."""
 
     def __init__(self, **kwargs: Any) -> None:
         self.code = kwargs.get("code", "")
         self.message = kwargs.get("message", "")
         self.details = kwargs.get("details", None)
-        self.status = LogsQueryStatus.FAILURE
```

Comment:
I think we still need the attr defined, can you double-check?

Reply:
Ah good catch! Re-added those back.